src/L1VotePool.sol

```diff
@@ -41,6 +42,7 @@ abstract contract L1VotePool {
   /// @param _governor The address of the L1 Governor contract.
   constructor(address _governor) {
     GOVERNOR = IGovernor(_governor);
+    console2.logAddress(address(GOVERNOR));
```

Comment:
leftover from debugging and needs to be removed?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea